.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Test launch_ros_sandbox
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  schedule:
+    # Run every hour
```

Comment:
Do we need to run these frequently?

Reply:
Yes, I'd like to get an autobuild every hour to detect dependencies breakage.